### ISSUE TITLE
Nexstrain CLI standalone installation fails on EC2 Amazon Linux 2

### ISSUE THREAD

**Issue by hannahhanle** [nextstrain/cli#234]
### Current Behavior
I wasn't able to install Nextstrain CLI using EC2 Amazon Linux 2. Please see the instance details below: 
- Platform: Amazon Linux 
- Platform details: Linux/ UNIX
- Description: Amazon Linux 2 Kernel 5.10 AMI 2.0.20221004.0 x86_64 HVM gp2
- AMI name: amzn2-ami-kernel-5.10-hvm-2.0.20221004.0-x86_64-gp2
- AMI ID: ami-09d3b3274b6c5d4aa

Running
```
curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/linux | bash
```
returns error: 
```
/home/ec2-user/.nextstrain/cli-standalone/nextstrain: /lib64/libc.so.6: version `GLIBC_2.27' not found (required by /home/ec2-user/.nextstrain/cli-standalone/nextstrain)
```

It turns out that Amazon Linux 2 has GLIBC_2.26 as one of its core packages (specified [here](https://repost.aws/questions/QUrXOioL46RcCnFGyELJWKLw/glibc-2-27-on-amazon-linux-2)), and as such you cannot upgrade.

### Expected behavior
Nextstrain CLI should install successfully. 

### How to reproduce
Steps to reproduce the current behavior:
1. Launch a new EC2 instance
2. Select instance type Amazon Linux 2 Kernel 5.10 AMI 2.0.20221004.0 x86_64 HVM gp2
3. Launch instance
4. Run `curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/linux | bash`
5. See error

### Possible solution
(optional)

### Your environment: if browsing Nextstrain online
 - Operating system:
 - Browser:

### Your environment: if running Nextstrain locally
 - Operating system:
 - Browser:
 - Version (e.g. `auspice 2.7.0`):

**Comment by tsibley** [comment]
Thanks for the detailed bug report. This is an unfortunate consequence of building the standalone installation binaries on Ubuntu 18.04, as that ships with glibc 2.27.

Ideally we'd build with an older glibc (e.g. on an older distro), but this isn't super straightforward because our CI host is GitHub Actions where Ubuntu 18.04 is the oldest supported Linux distro:

https://github.com/nextstrain/cli/blob/cf0f3486ec785b6f9f988696d6c30ede363a3494/.github/workflows/ci.yaml#L78-L95

and even that [will soon be deprecated](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/). There are ways around this with builds on containerized older distros, but I haven't had the time to set those up. (I've been kind of hoping that the plans referenced in the code above for PyOxidizer to gain isolated builder images would happen first. :-) 

In the meantime, an ok workaround is to install with Pip (as I understand from @victorlin that you've done :-):

    pip install nextstrain-cli

Or even Conda/Mamba (esp. if you're already using it).

    mamba install nextstrain-cli -c conda-forge -c bioconda --override-channels --strict-channel-priority

For posterity, here's the `pyoxizider analyze` output for the Nextstrain CLI 5.0.1 binary we produce, showing the minimum dynamic library versions (all part of libc or the LSB):

```
Shared Library Dependencies
===========================
ld-linux-x86-64.so.2
  OK - Library part of Linux Standard Base and present on most distros

libc.so.6
  OK - Library part of Linux Standard Base and present on most distros

libdl.so.2
  OK - Library part of Linux Standard Base and present on most distros

libgcc_s.so.1
  OK - Library part of Linux Standard Base and present on most distros

libm.so.6
  OK - Library part of Linux Standard Base and present on most distros

libpthread.so.0
  OK - Library part of Linux Standard Base and present on most distros

librt.so.1
  OK - Library part of Linux Standard Base and present on most distros

libutil.so.1
  OK - Library part of Linux Standard Base and present on most distros

Symbol Versioning
=================

gcc
-----

Minimum Version: 4.2.0
Minimum Distro Versions:
  Debian 7
  Fedora 16
  OpenSUSE 11.4
  RHEL 6
  Ubuntu 12.04

glibc
-----

Minimum Version: 2.27
Minimum Distro Versions:
  Debian 10
  Fedora 28
  OpenSUSE 15.3
  RHEL 8
  Ubuntu 18.04
```

**Comment by tsibley** [comment]
Noting that PyOxidizer (theoretically) [supports down to glibc 2.19](https://pyoxidizer.readthedocs.io/en/stable/pyoxidizer_packaging_python_distributions.html#binary-portability-of-distributions):

> On the glibc linked Linux distributions, they use an old glibc version for symbol versions, enabling them to run on Linux distributions created years ago. (The current version is 2.19, which was released in 2014.)

But we'd also have to run our `pyoxidizer build` invocation on a system with the minimum glibc version we want to target. 2.26 is old enough for Amazon Linux 2, for example, but not, say, CentOS 7 (which is still in use places and receiving active maintenance until Q2 2024!).

**Comment by tsibley** [comment]
In subsequent investigation, I found that PyOxidizer's standalone Python builds actually build with glibc 2.16 (despite the docs saying 2.19), which _is_ old enough for CentOS 7. So we can get there! See #243 for the changes to get to glibc 2.17.

**Comment by tsibley** [comment]
Standalone installation should work on Amazon Linux 2 [as of Nextstrain CLI 6.0.1](https://github.com/nextstrain/cli/blob/6.0.1/CHANGES.md#601-3-january-2023), which is going thru CI and the release process now.

**Comment by tsibley** [comment]
After an issue with the release of 6.0.1, this is now released in 6.0.2.
